tensorflow/lite/micro/kernels/quantize_test.cc

```diff
@@ -122,6 +122,8 @@ void TestRequantize(const int* input_dims_data, const float* input_data,
 TF_LITE_MICRO_TESTS_BEGIN
 
 #if !defined(XTENSA)
+#if !defined(CEVA_BX1) && !defined(CEVA_SP500)
+// No uint8 support in CEVA's optimized kernels
```

Comment:
Let's remove uint8 support from quantize. I can do this by Friday, or you can prepare a PR sooner if that is something that you would like to take on.

https://github.com/tensorflow/tensorflow/issues/47417